calico-vpp-agent/policy/policy_server.go

```diff
@@ -1677,50 +1674,49 @@ func (s *Server) createAllowFromHostPolicy() (err error) {
 }
 
 func (s *Server) createEndpointToHostPolicy( /*may be return*/ ) (err error) {
-	pol := &Policy{
+	workloadsToHostPolicy := &Policy{
 		Policy: &types.Policy{},
 		VppID:  types.InvalidID,
 	}
-	r_deny_workloads := &Rule{
+	action := types.ActionDeny
+	if s.getEndpointToHostAction() == "ACCEPT" {
+		action = types.ActionAllow
+	}
+	workloadsToHostRule := &Rule{
 		VppID: types.InvalidID,
 		Rule: &types.Rule{
-			Action: types.ActionDeny,
+			Action: action,
 		},
 		SrcIPSetNames: []string{"ipset1"},
 	}
-	ipset := NewIPSet()
-	ps := PolicyState{IPSets: map[string]*IPSet{"ipset1": ipset}}
-	err = ipset.Create(s.vpp)
-	if err != nil {
-		return err
-	}
-	pol.InboundRules = append(pol.InboundRules, r_deny_workloads)
-	err = pol.Create(s.vpp, &ps)
+	ps := PolicyState{IPSets: map[string]*IPSet{"ipset1": s.allPodsIpset}}
```

Comment:
`ipset1` ? Is this needed ? Just wondering whether this will conflict with another name, we could also leave it blank no ?

Reply:
it is just an identifier that should be the same as in the rule list of ipsets
The rule has the ipset named "ipset1", so it looks for this name in the policyState

Reply:
Agreed, but this name comes from Calico, if at some point Calico generates `ipset1` as an ipset name, both will conflict

Reply:
Also we could use something more descriptive than `ipset1` e.g. `calico-vpp-XXX-ipset` with `XXX` something like `wep-addr` ?

Reply:
fair